suite/refill/eth_refill_test.go

```diff
@@ -0,0 +1,84 @@
+package refill
+
+import (
+	"context"
+	"github.com/ethereum/go-ethereum/common"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/integrations-framework/client"
+	"github.com/smartcontractkit/integrations-framework/contracts"
+	"github.com/smartcontractkit/integrations-framework/suite"
+	"github.com/smartcontractkit/integrations-framework/tools"
+	"math/big"
+	"strings"
+	"time"
+)
+
+var _ = Describe("Eth refill suite", func() {
+	DescribeTable("Can work after refill", func(
+		initFunc client.BlockchainNetworkInit,
+		fluxOptions contracts.FluxAggregatorOptions,
+	) {
+		s, err := suite.DefaultLocalSetup(initFunc)
+		Expect(err).ShouldNot(HaveOccurred())
+		fluxInstance, err := s.Deployer.DeployFluxAggregatorContract(s.Wallets.Default(), fluxOptions)
+		Expect(err).ShouldNot(HaveOccurred())
+		err = fluxInstance.Fund(s.Wallets.Default(), big.NewInt(0), big.NewInt(1e18))
+		Expect(err).ShouldNot(HaveOccurred())
+		err = fluxInstance.UpdateAvailableFunds(context.Background(), s.Wallets.Default())
+		Expect(err).ShouldNot(HaveOccurred())
+		clNodes, nodeAddrs, err := suite.ConnectToTemplateNodes()
+		oraclesAtTest := nodeAddrs[:3]
+		clNodesAtTest := clNodes[:3]
+		Expect(err).ShouldNot(HaveOccurred())
+
+		// set oracles and submissions
+		err = fluxInstance.SetOracles(s.Wallets.Default(),
+			contracts.SetOraclesOptions{
+				AddList:            oraclesAtTest,
+				RemoveList:         []common.Address{},
+				AdminList:          oraclesAtTest,
+				MinSubmissions:     3,
+				MaxSubmissions:     3,
+				RestartDelayRounds: 0,
+			})
+		Expect(err).ShouldNot(HaveOccurred())
+		oracles, err := fluxInstance.GetOracles(context.Background())
+		Expect(err).ShouldNot(HaveOccurred())
+		log.Info().Str("Oracles", strings.Join(oracles, ",")).Msg("oracles set")
+
+		// set variable adapter
+		adapter := tools.NewExternalAdapter()
+
+		// Send Flux job to chainlink nodes
+		for _, n := range clNodesAtTest {
+			fluxSpec := &client.FluxMonitorJobSpec{
+				Name:              "flux_monitor",
+				ContractAddress:   fluxInstance.Address(),
+				PollTimerPeriod:   15 * time.Second, // min 15s
+				PollTimerDisabled: false,
+				ObservationSource: client.ObservationSourceSpec(adapter.InsideDockerAddr + "/variable"),
+			}
+			_, err = n.CreateJob(fluxSpec)
+			Expect(err).ShouldNot(HaveOccurred())
+		}
+		// set data, wait for round timeout, refill and check round is complete
```

Comment:
This test needs to have the steps of:
```
Add flux job > flux job runs > nodes run out of eth > eth refilled > see if job still works
```
Currently, it doesn't include the job running and waiting for the node to run out of ETH